modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-installing-operator-from-operatorhub-using-cli-{context}']
+= Installing from the OperatorHub using the CLI
+
+Instead of using the {product-title} web console, you can install an Operator
+from the OperatorHub using the CLI. Use the `oc` command as follows to install
+(subscribe to) an Operator for a selected namespace.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin`
+permissions.
+
+.Procedure
+
+. View the list of Operators available to the cluster from the OperatorHub:
++
+----
+$ oc get packagemanifests -n openshift-marketplace
+----
+
+. To identify the Operators to install, create a CatalogSourceConfig object YAML
+file (for example, `csc.cr.yaml`) containing information similar to the
+following:
+
++
+.Example CatalogSourceConfig
+[source,yaml]
+----
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "example"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "global" <1>
+  packages: "myoperator,another-operator" <2>
+----
+<1> Set the `targetNamespace` to identify the namespace where you want the
+Operator to be available.
+<2> Set `packages` to a comma-separated list of Operators to which you want to
+subscribe.
+
+. Install the Operator to the selected namespace:
++
+----
+$ oc apply -f csc.cr.yaml
+----
+
+. Later, if you want to install more Operators, update your CatalogSourceConfig
+file (in this example, `csc.cr.yaml`) with more packages. For example:
++
+.Example updated CatalogSourceConfig
+[source,yaml]
+----
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "example"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "global"
+  packages: "myoperator,another-operator,this-operator-too" <1>
+----
+<1> Add new packages to existing package list.
+
+. To update the CatalogSourceConfig object, run the `oc apply` command again:
++
+----
+$ oc apply -f csc.cr.yaml
+----
++
+The Operator generates a CatalogSource from your CatalogSourceConfig in the
+namespace specified in `targetNamespace`.
```

Comment:
Please be aware of using install and subscription separately here. All creating the catalogsourceconfig and catalogsource does is install the operator onto the cluster. And for now, correct me if I am wrong here, we should be requiring that all catalogsourceconfigs are installed to the openshift-marketplace namespace. Where they are created is unrelated to what namespace the operator is installed to.